plugins/es/qml/Keyboard_es_email.qml

```diff
@@ -85,8 +85,8 @@ KeyPad {
             LanguageKey    { id: languageMenuButton;                     anchors.left: symShiftKey.right; height: parent.height; }
             CharKey        { id: atKey;    label: "@"; shifted: "@";     anchors.left: languageMenuButton.right; height: parent.height; }
             SpaceKey       { id: spaceKey;                               anchors.left: atKey.right; anchors.right: urlKey.left; noMagnifier: true; height: parent.height; }
-            UrlKey         { id: urlKey; label: ".es"; extended: [".com", ".mx", ".co", ".ar", ".pe", ".ve", ".cl"]; anchors.right: dotKey.left; height: parent.height; }
-            CharKey        { id: dotKey;      label: "."; shifted: "."; extended: ["?", "-", "_", "!", "+", "%","#","/"]; extendedShifted: ["?", "-", "_", "!", "+", "%","#","/"]; anchors.right: enterKey.left; height: parent.height; }
+            UrlKey         { id: urlKey; label: ".com"; extended: [".com", ".mx", ".es", ".org", ".net"]; anchors.right: dotKey.left; height: parent.height; }
```

Comment:
Why have the following domains been removed?
.co -> Colombia
.ar -> Argentina
.pe -> Peru
.ve -> Venezuela
.cl -> Chile

And these have been left?
.es -> España
.mx -> Mexico

Reply:
no idea why the original author removed those, i'll add them back, seems relevant